### DISCUSSION TITLE
Team-Based Assignment

### DISCUSSION THREAD

**Discussion by YunusEmreAlps** [polonel/trudesk#540] [Q&A] [answered]
Hello,

I hope you're well and everything is good for you.

Does this platform have a team-based assignment?
Can I assign a ticket I created to another team?

![WhatsApp Image 2022-07-28 at 16 21 54](https://user-images.githubusercontent.com/44679555/181515662-3a5659e3-59f8-4bcd-b635-55cd72e19a38.jpeg)

As far as I understand, all agents are listed here. In this section, I want to list the teams.

Thank you very much.
Best regards,

**Comment by polonel** [comment, marked as answer]
Currently, Teams cannot be assigned. This is something that is already under active development. 

When it is complete every ticket will be assigned to a team and then a user within that team. (Or only assign to the team and leave the user null).

**Reply by YunusEmreAlps** [reply]
May I know when will this feature be added? and If I list the teams instead of the agent will this solve the problem quickly?

**Reply by polonel** [reply]
I don't have a firm ETA yet as a lot of systems are being rewritten from their original state back in 2015-2016. 

It's actually a bit more in-depth, the mongo documents are linked to access ObjectIDs from certain tables, and code is scattered through to query User Objects when looking at assignees.

ie:
https://github.com/polonel/trudesk/blob/master/src/models/ticket.js#L83-L85

**Comment by YunusEmreAlps** [comment]
I want to use this platform but this feature really important for me. The system is a little bit complex for me but if you want to I can help. 

I guess two dropdown lists solve this problem. The first one is used for the select team/group/agent and the second one is who is that team support for this problem.
